docs/en/setup/backend/static-gateways.md

```diff
@@ -0,0 +1,22 @@
+# Static Gateways/Proxies
+
+The word "static" here means that the gateways are not automatically/dynamically registered by SkyWalking agent when they are started,
+but configured statically in `gateways.yml` file or via [Dynamic Configuration](dynamic-config.md). The reason why they can't register
```

Comment:
`The reason of`, not `why`

Reply:
I remembered that this is not a grammar error, just like `the person who does...`, `the PR that changes...`, `the place where I live...`, `the way how we get here`, if you prefer I'll change to `The reason of that they can't register...`

Reply:
I search for a random webpage, seems that people say like this too https://www.oxfordlearnersdictionaries.com/definition/english/reason_1

Reply:
OK. I just feel a little strange, if it is right, keep it.